src/Employee.java

```diff
@@ -0,0 +1,47 @@
+public class Employee {
+    private final String fullName;
+    private int department;
+    private float salary;
+    private final int id;
+    static int counter = 0;
+
+    public static int id() {
+        counter++;
+        return counter;
+    }
+    public Employee (String fullName, int department, int salary) {
+        this.id = id();
+        this.fullName = fullName;
+        this.department = department;
+        this.salary = salary;
+    }
+
+    public String getFullName() {
+        return fullName;
+    }
+
+    public int getDepartment() {
+        return department;
+    }
+
+    public float getSalary() {
+        return salary;
+    }
+
+    public void setDepartment(int department) {
+        this.department = department;
+    }
+
+    public void setSalary(float salary) {
+        this.salary = salary;
+    }
+
+    public String toString() {
+        return this.id + " ФИО: " + this.fullName + ", отдел: " + this.department + ", зарплата: " +
+                this.salary;
+    }
+
+    public int toStringId() {
```

Comment:
```suggestion
    public int getId() {
```
геттеры называются на get и название поля с большой буквы